test/cryptography/sign.js

```diff
@@ -301,6 +302,19 @@ ${defaultSecondSignature}
 		});
 	});
```

Comment:
Could you duplicate the `signData` tests for `signDataWithPassphrase`? We need to check the exported function is working as intended.

Reply:
@willclarktech done